### PR TITLE
feature(common): my first commit [WEBSITES-3]

### PR DESCRIPTION
**Pull request by coyo97**
## sumary
updated the readme

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,9 +1,19 @@
 import Theme from "./Theme/Theme";
 import NavMenu from "./ui/components/navMenu/NavMenu";
+import FormClient from "./ui/components/formClient/FormClient";
```
Comment (Ditmar) [inline src/App.js:12]: please use simple quotes here

File: src/App.js [hunk 2]
```diff
@@ -1,9 +1,19 @@
...
+import FormDirect from "./ui/components/formClient/FormDirect";
```
Comment (Ditmar) [inline src/App.js:13]: please use simple quotes here

File: src/App.js [hunk 3]
```diff
@@ -1,9 +1,19 @@
...
+import FormFact from "./ui/components/formClient/FormFact";
```
Comment (Ditmar) [inline src/App.js:14]: please use simple quotes here

File: src/ui/components/formClient/FormClient.jsx [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+      errors.telefono = "El número de teléfono es obligatorio.";
+    }
+
+    // Validar el formato del correo electrónico
+    const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+    if (cliente.email && !emailRegex.test(cliente.email)) {
+      errors.email = "El correo electrónico no es válido.";
+    }
+    return errors;
+  };
```
Comment (Ditmar) [inline src/ui/components/formClient/FormClient.jsx:59]: All this function can be improve to this way
```
const validateForm = () => {
  // Realizar la validación del formulario
  let errors = {};

  // Mapa con los nombres de los campos y sus mensajes de error correspondientes
  const fieldNames = {
    id: "El ID es obligatorio.",
    nombre: "El nombre es obligatorio.",
    apellido: "El apellido es obligatorio.",
    email: "El correo electrónico es obligatorio.",
    telefono: "El número de teléfono es obligatorio."
  };

  // Validar los campos obligatorios
  Object.keys(fieldNames).forEach(key => {
    if (!cliente[key]) {
      errors[key] = fieldNames[key];
    }
  });

  // Validar el formato del correo electrónico
  const emailRegex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
  if (cliente.email && !emailRegex.test(cliente.email)) {
    errors.email = "El correo electrónico no es válido.";
  }
  return errors;
};
```

File: src/ui/components/formClient/FormDirect.jsx [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+    }));
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const validationErrors = validateForm();
+    if (Object.keys(validationErrors).length === 0) {
+      console.log(envio);
+      // Aquí puedes realizar alguna acción con los datos de envío, como guardarlos en una base de datos.
+    } else {
```
Comment (Ditmar) [inline src/ui/components/formClient/FormDirect.jsx:31]: remove the comment

File: src/ui/components/formClient/FormDirect.jsx [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+      ...prevEnvio,
+      [name]: value,
+    }));
+  };
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    const validationErrors = validateForm();
+    if (Object.keys(validationErrors).length === 0) {
+      console.log(envio);
```
Comment (Ditmar) [inline src/ui/components/formClient/FormDirect.jsx:29]: remove the console log, and use only english 'envio*'

File: src/ui/components/formClient/FormDirect.jsx [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+    if (!envio.codigoP) {
+      errors.codigoP = "La dirección 1 es obligatoria.";
+    }
+
+    return errors;
+  };
+
+  const [etiquetasSeleccionadas, setEtiquetasSeleccionadas] = useState([]);
+
+  const handleEtiquetaSeleccionada = (etiqueta) => {
```
Comment (Ditmar) [inline src/ui/components/formClient/FormDirect.jsx]: use only english

File: src/ui/components/formClient/formDireEnv.style.js [hunk 8]
```diff
@@ -0,0 +1,53 @@
...
+    fontFamily: "Inter-Regular",
+    [mq("xxs")]: {
+      width: "100%",
+      height: "100%",
+    },
+    [mq("md")]: {
+      width: "100%",
+      height: "100%",
+    },
+    border: "1px solid #000",
```
Comment (Ditmar) [inline src/ui/components/formClient/formDireEnv.style.js]: all colors must be used from the color palette

File: src/ui/components/formClient/formFact.style.js [hunk 9]
```diff
@@ -0,0 +1,40 @@
...
+    gridTemplateColumns: "repeat(2, 1fr)",
+    gridGap: "10px",
+    width: "100%",
+    justifyContent: "space-between",
+  },
+  input: {
+    width: "100%", // Asegura que los inputs ocupen el ancho completo del contenedor
+    padding: "8px", // Ajusta el padding para dar espacio entre los inputs y el borde del contenedor
+    borderRadius: "4px", // Agrega un borde redondeado a los inputs
+    border: "2px solid #ccc", // Añade un borde alrededor de los inputs
```
Comment (Ditmar) [inline src/ui/components/formClient/formFact.style.js]: remove the comments of the pr, and use the colors from the palette

**Review by Ditmar — CHANGES_REQUESTED** [review]
You need to use the english as a standar in all your code

**Comment by olmedo90** [comment]
image

It is not according to Figma. correct the following: (position of the titles, color of components, name of the inputs, border of the buttons)

**Comment by olmedo90** [comment]
image

It is according to Figma. continue with the same interest.👍

**Review by Ditmar — APPROVED** [review]
approved

**Pull request merged**